util/util.go

```diff
@@ -912,17 +912,20 @@ func TrimFilesystem(volumeName string) error {
 		return err
 	}
 
-	mountOutput, err := nsExec.Execute("bash", []string{"-c", fmt.Sprintf("mount | grep /dev/longhorn/%s | awk '{print $3}'", volumeName)})
+	mountOutput, err := nsExec.Execute("bash", []string{"-c", fmt.Sprintf("cat /proc/mounts | grep /dev/longhorn/%s | awk '{print $2}'", volumeName)})
 	if err != nil {
 		return fmt.Errorf("cannot find volume %v mount info on host: %v", volumeName, err)
 	}
 
 	mountList := strings.Split(strings.TrimSpace(mountOutput), "\n")
-	if len(mountList) != 1 {
-		return fmt.Errorf("zero or multiple mount points for volume %s: %+v", volumeName, mountList)
+
+	var mountpoint string
+	for _, m := range mountList {
+		mountpoint = m
```

Comment:
Should we check if this mount point is the valid one?

Reply:
Would be better. But how to check it is a valid one?

Reply:
Function `ensureMountPoint` in the pkg `csi` already implements that

Reply:
`ensureMountPoint` will try to clean up the corrupted mount point so it's not just a validation behavior, so probably we need to have a separate one for `mount.IsCorruptedMnt(err)` only.

https://github.com/longhorn/longhorn-manager/blob/db24c350211c524eab04afc702fdc0497aabeed6/csi/util.go#L216-L220

Reply:
I'm thinking if we need to check if the mountpoint is corrupted. If it is corrupted, the fstrim should return error.

Reply:
Maybe we can have a try.

Reply:
OK. It shows error `fstrim: vol: FITRIM ioctl failed: Input/output error`.

Here it's better to figure out which mount point is the valid one then apply `fstrim`

Reply:
Then, let me add a check.